.github/workflows/ruff.yaml

```diff
@@ -63,26 +76,45 @@ jobs:
         with:
           name: wheels
           path: dist
+      - name: "Archive binary"
+        run: |
+            ARCHIVE_FILE=ruff-aarch64-apple-darwin.tar.gz
+            tar czvf $ARCHIVE_FILE -C target/aarch64-apple-darwin/release ruff
+            shasum -a 256 $ARCHIVE_FILE > $ARCHIVE_FILE.sha256
```

Comment:
Do we need to strip the release binary, like in `ripgrep`?

https://github.com/BurntSushi/ripgrep/blob/44fb9fce2c1ee1a86c450702ea3ca2952bf6c5a7/.github/workflows/release.yml#L137

Reply:
The binary isn't that big, I wouldn't bother strip it just yet, besides strip symbols makes panic backtrace hard to read.

I'd use `maturin build --strip` to also strip the binary in wheels if we want to strip the release binaries.

Reply:
Okay cool, sounds good -- I wasn't sure.

Reply:
https://github.com/charliermarsh/ruff/blob/d8e709648d2fe7c2bd7cf867fda1918bf6f092dc/pyproject.toml#L47

Actually it's already using `maturin build --strip`.